tasks/get_record.json

```diff
@@ -1,17 +1,17 @@
 {
     "puppet_task_version": 1,
     "supports_noop": false,
-    "description": "Create a ServiceNow object",
-    "files": ["ruby_task_helper/files/task_helper.rb"],
+    "description": "Get a ServiceNow record",
     "remote": true,
+    "files": ["ruby_task_helper/files/task_helper.rb"],
     "parameters": {
         "table": {
             "description": "ServiceNow table",
-            "type": "Optional[String[1]]"
-        },
-        "data": {
-            "description": "ServiceNow data",
             "type": "String[1]"
+        },
+        "fields": {
+            "description": "ServiceNow fields. This is a hash of <field_name> => <value>",
```

Comment:
So what is `fields` here? Are those the fields to filter on when retrieving record? Would be nice to provide a bit more detail

Reply:
Also, I think maybe for this one, `fields` could be Optional. Seems weird to have to specify `fields` when getting a record.

Reply:
yeah in this case the fields are what gets used as a query on the table. If you don't specify anything I think you just get the entire table.

Reply:
changed to query_fields to be more clear